tests/test_modeling_common.py

```diff
@@ -1280,24 +1279,25 @@ def flatten_output(output):
                     )
 
                 # Test that the model can be serialized and restored properly
-                with tempfile.TemporaryDirectory() as tmp_dir_name:
-                    pkl_file_name = os.path.join(tmp_dir_name, "model.pkl")
-                    try:
-                        with open(pkl_file_name, "wb") as f:
-                            pickle.dump(traced_model, f)
-                        with open(pkl_file_name, "rb") as f:
-                            loaded = pickle.load(f)
-                    except Exception as e:
-                        self.fail(f"Couldn't serialize / deserialize the traced model: {e}")
-
-                    loaded_output = loaded(**filtered_inputs)
-                    loaded_output = flatten_output(loaded_output)
-
-                    for i in range(num_outputs):
-                        self.assertTrue(
-                            torch.allclose(model_output[i], loaded_output[i]),
-                            f"serialized model {i}th output doesn't match model {i}th output for {model_class}",
-                        )
+                # TODO: fix that if possible AND relevant.
```

Comment:
An issue should be opened to track this to make sure this work isn't forgotten

Reply:
Honestly, the real question is: is-it relevant to want to be able to pickle that. But yes, I created an issue (#30575)

Reply:
If it's not relevant anymore - then I'd be pro just cutting this whole commented out section

Reply:
Done